src/backend/web/handlers/index.py

```diff
@@ -189,3 +192,27 @@ def index_insights(template_values: Dict[str, Any]) -> str:
 @cached_public
 def about() -> str:
     return render_template("about.html")
+
+
+@cached_public
+def avatar_list(year: Optional[Year] = None) -> Response:
+    year = year or SeasonHelper.get_current_season()
+
+    valid_years = list(range(2018, SeasonHelper.get_max_year() + 1))
+    valid_years.remove(2021)  # No avatars in 2021 :(
+
+    if year not in valid_years:
+        abort(404)
+
+    avatars_future = Media.query(
+        Media.media_type_enum == MediaType.AVATAR, Media.year == year
+    ).fetch_async()
+    avatars = sorted(avatars_future.get_result(), key=lambda a: a.team_number)
+
+    template_values = {
+        "year": year,
+        "valid_years": valid_years,
```

Comment:
Mentioned before - but I don't think we need to pass `valid_years` to the template